### PR TITLE
Fix last-proxy-url and use functions

### PR DESCRIPTION
**Pull request by loicgasser**
Improve readability, and fix a small issue with last-proxy-url

### CODE REVIEW

**Comment by oterral** [comment]
jenkins test

**Comment by loicgasser** [comment]
It should be good now.

File: Makefile [hunk 1]
```diff
@@ -48,10 +48,10 @@ E2E_TARGETURL ?= https://mf-geoadmin3.dev.bgdi.ch
 DEPLOY_TARGET ?= dev
 LESS_PARAMETERS ?= -ru
 KEEP_VERSION ?= 'false'
-LAST_VERSION := $(shell if [ -f .build-artefacts/last-version ]; then cat .build-artefacts/last-version 2> /dev/null; else echo '-none-'; fi)
+LAST_VERSION = $(call lastvalue,version)
 VERSION := $(shell if [ '$(KEEP_VERSION)' = 'true' ] && [ '$(LAST_VERSION)' != '-none-' ]; then echo '$(LAST_VERSION)'; else date '+%y%m%d%H%M'; fi)
 GIT_BRANCH := $(shell if [ -f .build-artefacts/deployed-git-branch ]; then cat .build-artefacts/deployed-git-branch 2> /dev/null; else git rev-parse --symbolic-full-name --abbrev-ref HEAD; fi)
-GIT_LAST_BRANCH := $(shell if [ -f .build-artefacts/last-git-branch ]; then cat .build-artefacts/last-git-branch 2> /dev/null; else echo 'dummy'; fi)
+GIT_LAST_BRANCH = $(all lastvalue,git-branch)
```
Comment (oterral) [inline Makefile]: should be `call`

**Comment by loicgasser** [comment]
fixed

**Pull request merged**